src/ui/contexts/ApiContext.tsx

```diff
@@ -30,7 +30,7 @@ export const ApiContextProvider = ({ children }: React.PropsWithChildren<Partial
   const rpcUrl = searchParams.get('rpc');
   const [preferredEndpoint, setPreferredEndpoint] = useLocalStorage<string>(
     LOCAL_STORAGE_KEY.PREFERRED_ENDPOINT,
-    ROCOCO_CONTRACTS.rpc,
+    POP_NETWORK_TESTNET.rpc,
```

Comment:
Could use another, the only reason to pick Pop Network is because it support both ink! v6 and ink! v5